sdk/textanalytics/Azure.AI.TextAnalytics/src/TasksStateTasksDetails.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// TasksStateTasksDetails.
+    /// </summary>
+    [CodeGenModel("TasksStateTasksDetails")]
+    public partial class TasksStateTasksDetails
+    {
+        internal TasksStateTasksDetails(DateTimeOffset lastUpdateDateTime, string name, JobStatus status) : base(lastUpdateDateTime, name, status)
```

Comment:
what is the purpose of this constructor?

Reply:
removed